packages/types/src/sign-client/engine.ts

```diff
@@ -309,6 +315,7 @@ export interface EnginePrivate {
   onSessionRequest(params: {
     topic: string;
     payload: JsonRpcRequest<JsonRpcTypes.RequestParams["wc_sessionRequest"]>;
+    transportType?: RelayerTypes.MessageEvent["transportType"];
```

Comment:
same here

Reply:
session requests can be sent over link mode or i'm mistaken?

Reply:
you are right, i misread the function